### ISSUE TITLE
catkin error for missing dependency: ros-indigo-pcl-ros

### ISSUE THREAD

**Issue by deb0ch** [Parrot-Developers/slamdunk_ros#1]
Hi,

When trying to follow your README on my freshly installed Ubuntu 14.04 laptop, I bumped into a catkin error due to a missing package ros-ingigo-pcl-ros, solved by using:

``` shell
sudo apt-get install ros-indigo-pcl-ros
```

I don't know if it is installed by default on the S.L.A.M.dunk, but your README doesn't precise on which computer we have to follow it.

Same issue for rqt and rviz.

It might be useful to either add these to the prerequisite:

``` shell
sudo apt-get install build-essential git ros-indigo-rqt ros-indigo-rviz
```

or state that these commands **must** be run on the S.L.A.M.dunk

**Comment by Sarcasm** [comment]
Hello,

Thank you for the feedback.
After a clean install of the ROS tools I can effectively some missing pieces which will be fixed in the next release (soon).

I tested on Ubuntu 14.04, and the only package that required to be installed manually were `ros-indigo-rviz ros-indigo-rqt` and `ros-indigo-rqt-plot`.

`rviz` and `PCL` are already declared in the package.xml files, which mean, the `rosdep update` step of the [README's Install section](https://github.com/Parrot-Developers/slamdunk_ros/blob/10b15bf397c763689ba975b7a6a1ad6bb06d094a/README.md#install) will install them.

**Comment by Sarcasm** [comment]
This is fixed in the 1.0.1 update.
Thank you for reporting the issue.
